packages/sdk/README.md

```diff
@@ -780,6 +803,18 @@ client.products.deleteSku(productId, skuId, { force: true })
 client.products.listSkus(productId)
 ```
 
+#### [Import SKUs using CSV](https://docs.voucherify.io/reference/import-skus-using-csv)
+
+```javascript
+client.products.importSkusUsingCSV()
```

Comment:
not a huge fan of word `Using`. 
`client.products.importSkusUsingCSV` -> `client.products.importSkusCSV`

Reply:
And what about params? :)  File path? Product id?